[docs] Add call out for networkmanager workaround (#836)
rancher/rke2: docs/install/airgap.md

# Air-Gap Install

> **Important:** If your node has NetworkManager installed and enabled, [ensure the interface ignore workaround is applied.](https://docs.rke2.io/known_issues/#networkmanager)

Reviewer: ```suggestion
**Important:** If your node has NetworkManager installed and enabled, [ensure that it is configured to ignore CNI-managed interfaces.](https://docs.rke2.io/known_issues/#networkmanager)
```

Author: I will implement these commit suggestions manually.
